README.md

```diff
@@ -245,44 +170,23 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+>If you like this project please contact me.
 
```

Comment:
- [ ] [OPTIONAL] Great job! However, you can also consider writing this line here:

> Please give a ⭐️if you like this project

This is because giving a star on a GitHub project means that it is liked by the people, which always gives a good impression 😃